vcloud.yaml

```diff
@@ -1,11 +1,334 @@
-openapi: 3.0.0
+openapi: 3.0.1
 info:
   title: V-Cloud API Reference
-  version: 1.0.0
+  description:
+    The V-Cloud API is a collection of web services used to interact with  Voci's
+    Automatic Speech Recognition (ASR) engine. V-Cloud is accessible to any programming
+    language capable of performing HTTP `POST` and `GET` requests. The service is
+    hosted at `https://vcloud.vocitec.com`.
+  version: 1.9.0-2022.06.02
+servers:
+  - url: https://vcloud.vocitec.com/
+tags:
+  - name: Transcribe Request
+    description:
+      The `/transcribe` endpoint accepts `POST` requests containing an audio
+      file, a zip file containing multiple audio files, or an audio file from an external
+      URL. Successful `/transcribe` requests return a `requestid` to retrieve the results.
```

Comment:
`...or an audio file from an external URL.`

Change to:

`...or a URL pointing to an audio file from which Voci will download the audio data.`